pkg/http/api.go

```diff
@@ -1713,7 +1714,7 @@ func (a *api) onQueryState(reqCtx *fasthttp.RequestCtx) {
 		qresp.Results[i].Data = jsoniter.RawMessage(resp.Results[i].Data)
```

Comment:
please replace `jsoniter.RawMessage` with `json.RawMessage`. So we can completely remove the package `github.com/json-iterator/go`.